cmd/relay-gossip/node/relaynode.go

```diff
@@ -30,6 +32,8 @@ type GossipRelayConfig struct {
 	CertPath        string
 	Insecure        bool
 	DrandPublicGRPC string
```

Comment:
Should this be a slice too, if we are supporting slice for HTTP? I suppose it is not essential, at least the code can accommodate it if needed with just a few more lines of code in startGRPC.

Reply:
Can you clarify what you mean by accommodate it? Do you mean just use `DrandPublicGRPC[0]`?

Reply:
I meant supporting failover access to multiple GRPCs (just like for the HTTPs). But in the interest of time, we don't need to add this now.